src/common-utils/main.sh

```diff
@@ -456,13 +457,17 @@ fi
 
 # Optionally configure zsh and Oh My Zsh!
 if [ "${INSTALL_ZSH}" = "true" ]; then
+    if [ ! -f "${user_home}/.zprofile" ] || ! grep -Fxq 'source $HOME/.profile' "${user_home}/.zprofile" ; then
```

Comment:
is this $HOME variable supposed to be replaced or no?  It's in single quotes so I don't think it would

Reply:
> is this $HOME variable supposed to be replaced or no?

Nope, It should be saved without any expansion.